package-lock.json

```diff
@@ -12039,11 +12039,12 @@
 			}
 		},
 		"del": {
-			"version": "git+https://github.com/sindresorhus/del.git#9e7550be045525afbd45bd7d591f3f539d48ca50",
+			"version": "git+https://github.com/sindresorhus/del.git#6e23f6ade68dcf5d9fc6be704fdc1deb8f95bf33",
 			"from": "git+https://github.com/sindresorhus/del.git",
 			"dev": true,
 			"requires": {
-				"globby": "^10.0.0",
+				"@types/glob": "^7.1.1",
+				"globby": "^6.1.0",
```

Comment:
It's odd to see a package.lock change in this changeset.  If it was up to me, I'd probably revert this.  I'm guessing it might be an artifact from the package updates down downstream.  I'll leave it up to you whether you leave it or not.

Reply:
eee ya... that must have been from an npm install after merging in latest EDTR master...
didn't notice that change and did not mean to include in this...